aries-site/src/layouts/main/Layout.js

```diff
@@ -198,6 +204,9 @@ export const Layout = ({
                           )}
                           {children}
                         </ContentSection>
+                        {accessibilityInfo && (
```

Comment:
It seems like we may want the accessibility info within the content section.

I'm also wondering if in the future we want to move where the accessibility section is placed within the guidance page if we will run into issues?

Reply:
I can move up into the ContentSection. I think based on Julias work with the new design we are going to want consistency in having the accessibility section at the bottom of each page.

Reply:
It looks like the accessibility section is still outside of the content section, were there issues moving it within?